erpnext/patches/v13_0/exotel_integration_deprecation_warning.py

```diff
@@ -0,0 +1,10 @@
+import click
+
+
+def execute():
+
+	click.secho(
+		"Exotel integration is moved to a separate app and will be removed from ERPNext in version-14.\n"
+		"Please install the app to continue using the integration: https://github.com/frappe/exotel_integration",
+		fg="yellow",
+	)
```

Comment:
Send system notification / persistent banner on `exotel settings` for reconfiguring.